torch/_inductor/scheduler.py

```diff
@@ -3087,14 +3261,26 @@ def can_fuse(self, node1: BaseSchedulerNode, node2: BaseSchedulerNode) -> bool:
         del device2
 
         shared_data_score = self.score_fusion_memory(node1, node2)
-        if shared_data_score == 0:
```

Comment:
Unrelated to this pr, it's to fix this test error, https://www.internalfb.com/intern/test/844425111960859?ref_report_id=0

Reply:
The main change here is to modify the condition of calling loop_reordering from `shared_data_score == 0` to `shared_data_score < config.score_fusion_memory_threshold`. 

In the failed test (https://github.com/pytorch/pytorch/blob/main/test/inductor/test_loop_ordering.py#L425), the two pointwise nodes originally have a shared read data (the scaling factor, which is just a scalar). The `shared_data_score` is not `0` but smaller than `config.loop_ordering_after_fusion`. 

Before the change:
`shared_data_score > 0` -> won't `loop_reorder` -> can't fused because of `shared_data_score < config.score_fusion_memory_threshold`
After the change:
`shared_data_score > 0` -> `loop_reorder` (shared_data_score < config.score_fusion_memory_threshold`) -> get a larger `shared_data_score` -> fused